ark/utils/plot_utils.py

```diff
@@ -28,52 +68,21 @@ def plot_overlay(predicted_contour, plotting_tif, alternate_contour=None, path=N
             2D or 3D numpy array of imaging signal
         alternate_contour (numpy.ndarray):
             2D numpy array of labeled cell objects
-        path (str):
-            path to save the resulting image
-        show (bool):
-            whether or not to show plot
     """
 
+    # just plot the outlines
```

Comment:
We should move this functionality (creating just the outlines) to `save_segmentation_labels`, since we'll want to do this for every FOV. Instead lets require a plotting tif for this function